cocotb/monitors/__init__.py

```diff
@@ -68,17 +69,17 @@ class Monitor(object):
         callback (callable): Callback to be called with each recovered transaction
             as the argument. If the callback isn't used, received transactions will
             be placed on a queue and the event used to notify any consumers.
-        event (event): Object that supports a ``set`` method that will be called when
+        event (cocotb.triggers.Event): (Subclass of) Event that will be called when
```

Comment:
Not sure (subclass of) is interesting here - it's implied pretty much everywhere, and it's unlikely that someone would subclass event anyway, I think.

Reply:
Good point, I updated the text accordingly.